src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/ClaimantResponseDefendantNotificationHandler.java

```diff
@@ -90,6 +92,12 @@ public String getScenario(CaseData caseData) {
             }
         } else if (isClaimantRejectRepaymentPlan(caseData)) {
             return SCENARIO_AAA6_CLAIMANT_INTENT_REJECT_REPAYMENT_ORG_LTD_CO_DEFENDANT.getScenario();
+        } else if (
+            !featureToggleService.isJudgmentOnlineLive()
+                && !NO.equals(caseData.getApplicant1Represented())
```

Comment:
I think you can change this to
`&& caseData.isApplicantLiP()`

Reply:
Updated from review